packages/integrations/vercel/src/edge/adapter.ts

```diff
@@ -27,6 +27,15 @@ function getAdapter(): AstroAdapter {
 		name: PACKAGE_NAME,
 		serverEntrypoint: `${PACKAGE_NAME}/entrypoint`,
 		exports: ['default'],
+		supportedAstroFeatures: {
+			hybridOutput: 'Stable',
+			staticOutput: 'Stable',
+			serverOutput: 'Stable',
+			assets: {
+				supportKind: 'Unsupported',
+				isSharpCompatible: true,
+			},
```

Comment:
On Vercel Edge, the only image service supported is Vercel's (when using `vercel({imageService: true})`)

Reply:
```suggestion
			assets: {
				supportKind: 'Stable',
				isSharpCompatible: true,
				isSquooshCompatible: false,
			},
```